release-notes/opensearch-release-notes-2.10.0.md

```diff
@@ -0,0 +1,687 @@
+* A new IP2Geo processor accesses location data based on IP addresses using external databases, allowing OpenSearch to enrich data with up-to-date geographical location information.
+* With  this release, OpenSearch Dashboards is upgraded to version 18 of Node.js, with backward-compatible support for Node.js versions 14 and 16. Users have the flexibility to choose from these versions to run the web application. As a result of this update, the Docker images for OpenSearch and OpenSearch Dashboards now use Amazon Linux 2023 as the base image, changed from Amazon Linux 2.
+
+### Experimental Features
+
+OpenSearch 2.10.0 includes the following experimental features. Experimental features are disabled by default. For instructions on how to enable them, refer to the documentation for the feature.
+
+* Build conversational search applications with new tools as part of ML Commons, including new APIs that enable conversational search and conversational memory as well as integrations with search pipelines that allow the use of conversational memory and large language models (LLMs) to answer questions.
+* Concurrent segment search gives you the option to query index segments in parallel at the shard level. This can deliver improved latency for many types of search queries.
+
+## Release Details
+[OpenSearch and OpenSearch Dashboards 2.10.0](https://opensearch.org/versions/opensearch-2-10-0.html) includes the following features, enhancements, bug fixes, infrastructure, documentation, maintenance and refactoring updates.
+
+OpenSearch [Release Notes](https://github.com/opensearch-project/OpenSearch/blob/2.10/release-notes/opensearch.release-notes-2.10.0.md).
+
+OpenSearch Dashboards [Release Notes](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/2.10/release-notes/opensearch-dashboards.release-notes-2.10.0.md).
+
+## FEATURES
+
+### OpenSearch Alerting
+* Add workflowIds field in getAlerts API  ([#1014](https://github.com/opensearch-project/alerting/pull/1014))
+* Add alertId parameter in get chained alert API and paginate associated alerts if alertId param is mentioned ([#1071](https://github.com/opensearch-project/alerting/pull/1071))
+* Chained Alert Behaviour Changes ([#1079](https://github.com/opensearch-project/alerting/pull/1079))
+
+
+### OpenSearch Alerting Dashboards
+* Updates for dark mode theme ([#695](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/695))
+* Add default Notification subject  ([#701](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/701))
+
+
+### OpenSearch Common Utils
+* Common utils to support Microsoft teams in notifications ([#428](https://github.com/opensearch-project/common-utils/pull/428))
+* Support list of monitor ids in Chained Monitor Findings ([#514](https://github.com/opensearch-project/common-utils/pull/514))
+
+
+### OpenSearch Custom Codecs
```

Comment:
Is this correct? We released ZSTD in 2.9.0 right ? We moved it to plugin in 2.10.0.

Reply:
I believe that is grabbed from here https://github.com/opensearch-project/custom-codecs/blob/main/release-notes/opensearch-custom-codecs.release-notes-2.10.0.0.md

Reply:
Does it makes sense to say "ZSTD has been moved to a plugin" ?